src/postmate.js

```diff
@@ -254,10 +254,13 @@ class Postmate {
     container = typeof container !== 'undefined' ? container : document.body, // eslint-disable-line no-use-before-define
     model,
     url,
+    iframeAllowedProperties = '',
   }) { // eslint-disable-line no-undef
     this.parent = window
     this.frame = document.createElement('iframe')
-    this.frame.allow = ''
+    if (iframeAllowedProperties) {
+      this.frame.allow = iframeAllowedProperties
+    }
```

Comment:
as @BartoszGamza have said it does not make sens to keep both: default value and if statement for `iframeAllowedProperties` property

Reply:
I think this check here is better than assigning empty string to this property though 👍